src/ui/control/navigation_control.js

```diff
@@ -15,6 +15,9 @@ const defaultOptions = {
  * A `NavigationControl` control contains zoom buttons and a compass.
  *
  * @implements {IControl}
+ * @param {Object} [options]
+ * @param {Object} [options.showCompass=true] If `false` will omit the compass button from the Navigation Control. The default is `true` to show a compass button.
```

Comment:
Small edit:
> @param {boolean} [options.showCompass=true] If `true`, the compass button is included.
> @param {boolean} [options.showZoom=true] If `true`, the zoom-in and zoom-out buttons are included.

Note the type should be marked `{boolean}`, not `{Object}`.

  The default value is shown in the generated docs, so no need to state it.

Reply:
Fixed.